### PR TITLE
Document RPC authentication JWT algorithm option

### PR DESCRIPTION
**Pull request by JorgeFPastor**
When [generating a keypair for JWT public key authentication](https://besu.hyperledger.org/public-networks/how-to/use-besu-api/authenticate#jwt-public-key-authentication), if you use a different method than `RS256` to generate such keypair, especially if you use a `ESXXX` method, you also need to specify such method in the command line options. Otherwise you'll face a `java.security.InvalidKeyException: Invalid RSA public key` exception when running `besu`.

### CODE REVIEW

File: versioned_docs/version-23.10.2/public-networks/how-to/use-besu-api/authenticate.md
Comment (macfarla) [inline versioned_docs/version-23.10.2/public-networks/how-to/use-besu-api/authenticate.md:1]: we don't want to update version 23.10.2 of the doc, only current

File: versioned_docs/version-23.10.2/public-networks/reference/cli/options.md
Comment (macfarla) [inline versioned_docs/version-23.10.2/public-networks/reference/cli/options.md:1]: we don't want to update version 23.10.2 of the doc, only current

File: versioned_docs/version-23.10.3/public-networks/how-to/use-besu-api/authenticate.md
Comment (macfarla) [inline versioned_docs/version-23.10.3/public-networks/how-to/use-besu-api/authenticate.md:1]: we don't want to update version 23.10.2 of the doc, only current

File: versioned_docs/version-23.10.3/public-networks/reference/cli/options.md
Comment (macfarla) [inline versioned_docs/version-23.10.3/public-networks/reference/cli/options.md:1]: we don't want to update version 23.10.2 of the doc, only current

File: docs/public-networks/how-to/use-besu-api/authenticate.md
Comment (macfarla) [inline docs/public-networks/how-to/use-besu-api/authenticate.md:1]: change in this file looks good

File: docs/public-networks/reference/cli/options.md
Comment (macfarla) [inline docs/public-networks/reference/cli/options.md:1]: change in this file looks good

**Review by macfarla — COMMENTED** [review]
the main change looks sensible to me. main comment though is we only want to change the current version of the docs, not versioned content.

**Review by alexandratran — APPROVED** [review]
Made some edits. LGTM.

**Pull request merged**